lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  afterEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { mountWithContext } from '../../../../test/bigtest/helpers';
+import {
+  ProfilesLabelInteractor,
+  ProfilesPopoverInteractor,
+} from './interactors';
+import ProfilesLabel from '../ProfilesLabel';
+
+import translations from '../../../../translations/stripes-data-transfer-components/en';
+
+describe('ProfilesLabel', () => {
+  const profilesLabel = new ProfilesLabelInteractor();
+  const popover = new ProfilesPopoverInteractor();
+
+  describe('rendering ProfileLabel', () => {
+    const link = 'http://localhost/';
+    const content = <div data-profiles-label-content>Content</div>;
+
+    beforeEach(async () => {
+      const overlayContainer = document.createElement('div');
+
+      overlayContainer.id = 'OverlayContainer';
+      document.body.append(overlayContainer);
+
+      await mountWithContext(
+        <ProfilesLabel
+          link={link}
+          content={content}
+        />
+      );
+    });
+
+    afterEach(() => {
+      document.getElementById('OverlayContainer').remove();
+    });
+
+    it('should display ProfileLabel component', () => {
+      expect(profilesLabel.isPresent).to.be.true;
+    });
+
+    it('should display the correct label text', () => {
+      expect(profilesLabel.text).to.equal(translations.profiles);
+    });
+
+    it('should display info icon button', () => {
+      expect(profilesLabel.infoButton.isPresent).to.be.true;
+    });
+
+    describe('clicking on info icon button', () => {
+      beforeEach(async () => {
+        await profilesLabel.infoButton.click();
+        await new Promise(resolve => { setTimeout(() => resolve(), 1000); });
+      });
+
+      it('should display put content', () => {
+        expect(popover.content.isPresent).to.be.true;
+      });
+
+      it('should apply correct href property', () => {
+        expect(popover.linkHref).to.equal(link);
+      });
+
+      it('should display put content', () => {
```

Comment:
Let's name for better clarity. Same [here](https://github.com/folio-org/stripes-data-transfer-components/pull/23/files#diff-af8755bd52a3d965735846c6f4fefd1cR63).
```suggestion
      it('should display passed in content', () => {
```